src/Symfony/Component/Notifier/Bridge/Pushover/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "symfony/pushover-notifier",
```

Comment:
please use 4 spaces indentation for this file

Reply:
done